kong.conf.default

```diff
@@ -1124,6 +1124,27 @@
                                  # Detailed discussion of this behavior is
                                  # available in the online documentation.
 
+#pg_connection_keepalive_timeout = 60000 # Specify the maximal idle timeout ( in ms )
```

Comment:
Personally, I'd prefer `pg_keepalive_timeout`, so do `pg_pool_size` and `pg_backlog`. This aligns with [pgmoon options](https://github.com/leafo/pgmoon/blob/master/README.md#newoptions).

Reply:
Yea makes sense, updated